files/en-us/web/css/css_flexible_box_layout/controlling_ratios_of_flex_items_along_the_main_ax/index.html

```diff
@@ -181,7 +181,7 @@ <h3 id="What_sets_the_base_size_of_the_item">What sets the base size of the item
 
 <ol>
  <li>Is <code>flex-basis</code> set to <code>auto</code>, and does the item have a width set? If so, the size will be based on that width.</li>
- <li>Is <code>flex-basis</code> set to <code>auto</code> or <code>content</code> (in a supporting browser)? If so, the size is based on the item size.</li>
+ <li>Is <code>flex-basis</code> set to <code>auto</code> or <code>content</code> (in a supporting browser)? If so, the size is based on the item's max-content size.</li>
```

Comment:
```suggestion
 <li>Is <code>flex-basis</code> set to <code>auto</code> or <code>content</code> (in a supporting browser)? If so, the size is based on the item's content size.</li>
```

It is pretty much max-content but not quite, so I think it's probably better to say it is the size of the content, rather than declaring it definitely max-content (https://www.w3.org/TR/css-flexbox-1/#valdef-flex-basis-content)

Reply:
That's a very precise reference, thank you.
Yes, the documentation says that it isn't exactly the max-content, but I thought that it was still appropriated to use this analogy as a means of being more didactic. This perception reinforced by the fact that the default flex-basis size was said to be, more than one time, the max-content size in this same article.
If the more descriptive sections of this article already made this didactical-licensed omission, don't you think that the summary one, which by definition should be even more concise, shouldn't also do it?
If you find that it's imprecise to use this didactic omission, do you think that it would be correct to adjust the same cases in this article?
Thank you in advance for your time and attention (:

Reply:
the reference I pointed to is the specification for flexbox, as you can see it states that this isn't quite max-content size, so I think referring to the content size (as in my edit) would be the best thing to do here. Feel free to raise an issue about other mentions.